### PR TITLE
feat(typescript): added support for typescript

### PR DESCRIPTION
**Pull request by Aman-Codes**
Added support for typescript

### CODE REVIEW

**Comment by Aman-Codes** [comment]
It still says that typescript is not supported so can someone guide me that what am I missing

**Comment by GMishx** [comment]
Hello @Aman-Codes , thanks for your contribution. But looking at TypeScript, the commenting styles looks exactly like CPP.
Can you try adding `.ts => cpp` to the `langMap` and see if it gives good results instead of creating a new class specific for ts?

**Comment by GMishx** [comment]
Sorry, I just realised the `lang` is also reported in the output. It will require a separate class to fix.

I just checkout your code and works fine for me. Can you please try to reinstall the code and try again?

Also, can you please add a test file for TS in the `testScript.py` and a corresponding test case in `nirjas/languages/tests/` folder? It will be great help.

**Comment by Aman-Codes** [comment]
@GMishx Added the test file code

**Comment by Aman-Codes** [comment]
Should I add code for other languages in this pull request only or one pull request per language ?

File: nirjas/languages/tests/test_typescript.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import unittest
```
Comment (GMishx) [inline nirjas/languages/tests/test_typescript.py:1]: Can you please add your copyright header here with LGPL license?

**Review by GMishx — APPROVED** [review]
Changes looks good. Works with sample ts files.

**Comment by GMishx** [comment]
> Should I add code for other languages in this pull request only or one pull request per language ?

It will be better to have a separate PR per language.

**Comment by Aman-Codes** [comment]
> It will be better to have a separate PR per language.

okay will add them in other pr

**Pull request merged**